install.sh

```diff
@@ -54,6 +59,18 @@ fatal() {
     exit 1
 }
 
+# check_target_mountpoint return success if the target directory is on a dedicated mount point
+check_target_mountpoint() {
```

Comment:
Should we instead fallback to fstab inspection iff `mountpoint -q /usr/local` fails (or the tool isn't present)?

Reply:
I'm fine with just using mountpoint directly, now that I know about it. I think we can expect util-linux or the busybox applet equivalent to exist pretty much everywhere.